### PR TITLE
Adding new SDK changes for NzN

### PR DESCRIPTION
**Pull request by hdornadula**
# Pull Request Template

## Description

This bumps the SDK GA version from 3.40.1 to 3.41.0 and preview version from 3.41.0-preview.1 to 3.42.0-preview.0. There were no changes to the GA public contract.

image

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Preview features are treated as a separate branch and will not be included in th
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### <a name="3.42.0-preview.0"/> [3.42.0-preview.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.42.0-preview.0) - 2024-06-07
+### <a name="3.41.0"/> [3.41.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.41.0) - 2024-06-07
+
+#### Added
+- [4489](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4489) Query: Adds DOCUMENTID extension method for LINQ
```
Comment (kirankumarkolli) [inline changelog.md:20]: Aditya please share the hotfix details to include also.

**Review by kirankumarkolli — APPROVED** [review]

File: changelog.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Preview features are treated as a separate branch and will not be included in th
...
+
 ### <a name="3.41.0-preview.1"/> [3.41.0-preview.1](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.41.0-preview.1) - 2024-06-06
 ### <a name="3.40.1"/> [3.40.1](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.40.1) - 2024-06-06
```
Comment (adityasa) [inline changelog.md:23]: @kirankumarkolli  - should we remove this if we are abandoning the hotfix?

Comment (hdornadula) [reply changelog.md]: Removed the extra content.

**Review by kirankumarkolli — APPROVED** [review]

**Review by adityasa — APPROVED** [review]
:shipit:

**Pull request merged**